.env

```diff
@@ -1,5 +1,8 @@
-REACT_APP_KEYCLOAK_URL=https://kbss.felk.cvut.cz:10808/auth
-REACT_APP_KEYCLOAK_REALM=kodi-dev
-REACT_APP_KEYCLOAK_CLIENTID=kodi-termit-ui-dev
-REACT_APP_REPORT_BUG_LINK=https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=bug&template=po-adavek-na-opravu.md
-REACT_APP_REQUEST_FEATURE_LINK=https://github.com/opendata-mvcr/sgov-assembly-line/issues/new?labels=enhancement&template=po-adavek-na-novou-funkcionalitu.md
+# React Environment Variables
+# https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables#expanding-environment-variables-in-env
+
+REACT_APP_CONTEXT=development
+REACT_APP_ID=kodi-termit-ui-dev
+REACT_APP_URL=http://localhost:3000
+REACT_APP_COMPONENTS=c2dvdlNlcnZlcjoNCiAgbmFtZTogU0dvViBTZXJ2aWNlDQogIHVybDogaHR0cHM6Ly9rYnNzLmZlbGsuY3Z1dC5jei9zZ292LXNlcnZlcg0KICBtZXRhOiB7fQ0KZGJTZXJ2ZXI6DQogIG5hbWU6IEEgZ3JhcGggZGF0YWJhc2UNCiAgdXJsOiBodHRwczovL2V4YW1wbGUuY29tL3NwYXJxbA0KICBtZXRhOiB7fQ0KYXV0aFNlcnZlcjoNCiAgbmFtZTogT0lEQyBBdXRoZW50aWNhdGlvbiBTZXJ2aWNlDQogIHVybDogaHR0cHM6Ly9rYnNzLmZlbGsuY3Z1dC5jejoxMDgwOC9hdXRoL3JlYWxtcy9rb2RpLWRldg0KICBtZXRhOiB7fQ0Kb250b2dyYXBoZXI6DQogIG5hbWU6IG9udG9HcmFwaGVyDQogIHVybDogaHR0cHM6Ly9zbG92bsOtay5nb3YuY3ovbW9kZWx1amVtZS92LW7DoXN0cm9qaS9vbnRvZ3JhcGhlcg0KICBtZXRhOg0KICAgIHdvcmtzcGFjZVBhdGg6ICc/d29ya3NwYWNlPSVXT1JLU1BBQ0VfSVJJJScNCnRlcm1pdFNlcnZlcjoNCiAgbmFtZTogVGVybUlUIGJhY2tlbmQNCiAgdXJsOiBodHRwczovL2tic3MuZmVsay5jdnV0LmN6L3Rlcm1pdC1zZXJ2ZXIta29kaQ0KICBtZXRhOiB7fQ0KdGVybWl0Og0KICBuYW1lOiBUZXJtSVQNCiAgdXJsOiBodHRwczovL3Nsb3Zuw61rLmdvdi5jei9tb2RlbHVqZW1lL3YtbsOhc3Ryb2ppL3Rlcm1pdA0KICBtZXRhOg0KICAgIHdvcmtzcGFjZVBhdGg6ICcjdm9jYWJ1bGFyaWVzP3dvcmtzcGFjZT0lV09SS1NQQUNFX0lSSSUnDQptaXNzaW9uQ29udHJvbDoNCiAgbmFtZTogTWlzc2lvbiBDb250cm9sDQogIHVybDogaHR0cHM6Ly9zbG92bsOtay5nb3YuY3ovbW9kZWx1amVtZS8NCiAgbWV0YToge30NCmlzc3VlVHJhY2tlcjoNCiAgbmFtZTogSXNzdWUgVHJhY2tlcg0KICB1cmw6IGh0dHBzOi8vZ2l0aHViLmNvbS9vcGVuZGF0YS1tdmNyL3Nnb3YtYXNzZW1ibHktbGluZS9pc3N1ZXMNCiAgbWV0YToNCiAgICBuZXdCdWc6IGh0dHBzOi8vZ2l0aHViLmNvbS9vcGVuZGF0YS1tdmNyL3Nnb3YtYXNzZW1ibHktbGluZS9pc3N1ZXMvbmV3P2xhYmVscz1idWcmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW9wcmF2dS5tZCZ0aXRsZT0NCiAgICBuZXdGZWF0dXJlOiBodHRwczovL2dpdGh1Yi5jb20vb3BlbmRhdGEtbXZjci9zZ292LWFzc2VtYmx5LWxpbmUvaXNzdWVzL25ldz9sYWJlbHM9ZW5oYW5jZW1lbnQmdGVtcGxhdGU9cG8tYWRhdmVrLW5hLW5vdm91LWZ1bmtjaW9uYWxpdHUubWQmdGl0bGU9DQo=
```

Comment:
Should not we generate inside each of these project repos a dedicated 'components' variable containing only the relevant services (not the whole sgov assembly line stack)?

Reply:
I thought about that and I think that we should keep the structure of components identical everywhere.

The best course of action would be to actually extract this information from the front end repositories and bundle all the common parts to a single dependency shared by all the apps. So that we'd have one repo that would define the components structure, type, and possibly the default (development) value as well. What we could (and probably should) extract is not just the common configuration, but potentially also authentication etc. Right now quite a lot of stuff is repeated across all the apps.

At the same time, if we don't extract the information - then we can do it as you suggest and essentially let the apps define it's interface (at least regarding components). We'd still end up with bunch of redundant information though.

I guess that it all comes down to whether we want the front end apps to be prototypes or production-ready :-)